htdocs/societe/admin/societe.php

```diff
@@ -236,6 +237,46 @@
 	}
 }
 
+// Disable Parent company
+if ($action == "setdisableParentCompagny") {
+	$val = GETPOST('value', 'int');
+	$res = dolibarr_set_const($db, "SOCIETE_DISABLE_PARENTCOMPANY", $val, 'yesno', 0, '', $conf->entity);
+
+	if (!($res > 0)) {
+		$error++;
+	} else {
+		$res = dolibarr_set_const($db, "SOCIETE_DISABLE_SUBSIDIARIES", $val, 'yesno', 0, '', $conf->entity);
```

Comment:
We should modify a second option when changing value of a first one. I think you can remove this. A fix was done in v16 so SOCIETE_DISABLE_SUBSIDIARIES is used only if SOCIETE_DISABLE_PARENTCOMPANY is set so you should no more need to modify it when changing SOCIETE_DISABLE_PARENTCOMPANY

Reply:
@eldy Ok i delete